exchange_server/datadog_checks/exchange_server/data/conf.yaml.example

```diff
@@ -23,4 +27,31 @@ instances:
     ##               rate, histogram or counter
     #
     # additional_metrics:
-    #   - ['MSExchange Content Filter Agent', none, 'Messages that Bypassed Scanning', exchange.content_filter.bypassed_messages, gauge]
+    #   - [MSExchange Content Filter Agent, none, Messages that Bypassed Scanning, exchange.content_filter.bypassed_messages,
+    #     gauge]
+
+## Log Section
+##
+## type - required - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - required - Set port if type is tcp or udp.
+##                                         Set path if type is file.
+##                                         Set channel_path if type is windows_event.
+## source  - required - Attribute that defines which Integration sent the logs.
+## encoding - optional - For file specifies the file encoding, default is utf-8, other
+##                       possible values are utf-16-le and utf-16-be.
+## service - optional - The name of the service that generates the log.
+##                      Overrides any `service` defined in the `init_config` section.
+## tags - optional - Add tags to the collected logs.
+##
+## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
+#
+# logs:
+#   - type: file
+#     path: C:\Program Files\Microsoft\Exchange Server\V15\TransportRoles\Logs\CommonDiagnosticsLog\*
```

Comment:
That should be two `\\` here right?

Reply:
I added the two `\\` in the spec.yaml, but it seems that `conf.yaml` removed them after being processed.

Reply:
Then add 4 `\\\\` in the spec.yaml :)

Reply:
Woah that's a game changer